cmd/gds/main.go

```diff
@@ -704,11 +704,19 @@ func initClient(c *cli.Context) (err error) {
 		return cli.NewExitError(err, 1)
 	}
 	client = api.NewTRISADirectoryClient(cc)
+	return nil
+}
 
+func initAdminClient(c *cli.Context) (err error) {
 	// Connect the admin client
 	if adminClient, err = admin.New(c.GlobalString("admin-endpoint")); err != nil {
 		return cli.NewExitError(err, 1)
 	}
+
+	// Attempt to login the admin client
+	if err = adminClient.Login(context.Background()); err != nil {
+		return cli.NewExitError(err, 1)
+	}
```

Comment:
Eventually I'd like to have profiles to make it easier to log into to vaspdirectory.net vs trisatest.net or to develop locally. This is a start toward that.